composer.json

```diff
@@ -12,7 +12,7 @@
         "phpunit/phpunit": "^5||^6.0"
     },
     "require-dev": {
-        "openeuropa/code-review": "^0.2",
+        "openeuropa/code-review": "^0.3",
```

Comment:
We don't need to state that we require 0.3 if we are not using the new features that are added in that version. We are still compatible with 0.2. A good idea would be to change the version range to `~0.2` then we are compatible with both 0.2, 0.3 and future versions.